src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader.php

```diff
@@ -60,6 +61,19 @@ public function load(mixed $resource, ?string $type = null): mixed
         // Force load ContainerConfigurator to make env(), param() etc available.
         class_exists(ContainerConfigurator::class);
 
+        $autoloaderRegistered = false;
+        if (null === $this->configBuilderAutoloader) {
+            $this->configBuilderAutoloader = function (string $class) {
+                if (!$this->generator || !str_starts_with($class, 'Symfony\\Config\\') || !str_ends_with($class, 'Config')) {
+                    return;
+                }
+
+                $this->configBuilder($class);
```

Comment:
The condition is easier to read in the affirmative way.
```suggestion
                if ($this->generator && str_starts_with($class, 'Symfony\\Config\\') && str_ends_with($class, 'Config')) {
                    $this->configBuilder($class);
                }
```